portality/forms/validate.py

```diff
@@ -542,6 +542,16 @@ def __call__(self, form, field):
         except Exception:
             raise validators.ValidationError(self.message)
 
+class Year(object):
+    def __init__(self, value, message=None):
+        self.value = value
+        self.message = message or "Date must be a year in 4 digit format (eg. 1987)"
+
+    def __call__(self, form, field):
+        if not field.data:
+            return
+        return 1900 <= field.data & field.data >= datetime.now().year
```

Comment:
python uses `and` - also, will field.data be a string or an int?

Reply:
this should use the new config key MINIMAL_OA_START_DATE

Reply:
no need for `and` anyway, python supports the syntax e.g. `0 <=x >= 100`

Reply:
`field.data` is int. I'm changing this as suggested.